playbooks/install_stack.yaml

```diff
@@ -384,6 +386,14 @@
         - /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovn-sriov.yaml
     when: sriov_interface is defined
 
+  - name: Add ovsdpdk to enabled services
+    set_fact:
+      service_envs: "{{ service_envs | union(ovsdpdk_env) }}"
+    vars:
+      ovsdpdk_env:
+        - /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovs-dpdk.yaml
```

Comment:
we should have two blocks here, one for when using OVS and another one when using OVN.

on https://github.com/shiftstack/dev-install/pull/148 I'm adding `ovn_enabled` param, we could probably use it here.